backend/repos/query.go

```diff
@@ -23,34 +31,35 @@ type QueryRepo struct {
 
 // NewQueryRepo creates a query repo
 func NewQueryRepo(db *sqlx.DB) (*QueryRepo, error) {
-	r := &QueryRepo{}
-	db.Mapper = reflectx.NewMapperFunc("json", strings.ToLower)
+	r := &QueryRepo{db: db}
 	_, err := db.Exec(`CREATE TABLE IF NOT EXISTS queries (
-		id TEXT PRIMARY KEY NOT NULL
-		, content TEXT NOT NULL
-		, owner_id TEXT NOT NULL
-		, active INTEGER NOT NULL
-		, description TEXT NOT NULL
-		, CONSTRAINT id_unique UNIQUE (id)
+        id TEXT PRIMARY KEY NOT NULL,
+        content TEXT NOT NULL,
+        owner_id TEXT NOT NULL,
+        active INTEGER NOT NULL,
+        description TEXT NOT NULL,
+        tags TEXT DEFAULT '[]',
```

Comment:
Create a new table with tags linked to queries instead of adding to the queries table